packages/web-app-files/src/components/AppBar.vue

```diff
@@ -27,9 +27,9 @@
             <oc-button
               id="new-file-menu-btn"
               key="new-file-menu-btn-enabled"
+              v-oc-tooltip="_cannotCreateDialogText"
```

Comment:
This one is showing `You have no permission to upload!` although I do have permissions to upload.

Reply:
I think the button needs some aria prop for the error messages in `_cannotCreateDialogText`

Reply:
I'll add an aria-label, the fact that you get the error message even though you're able to upload should perhaps be opened in another issue? :)

Reply:
Had a look into it and we should fix it in this PR since it's os prominent (and obviously broken 🐞 )

Reply:
Will be solved in the design system, no need to wait for it with this PR, since it's not going to master.